lib/carto/named_maps/api.rb

```
@@ -190,7 +190,7 @@ def log_response(response, action)
         # Hotfix for preventing https://rollbar.com/carto/CartoDB/items/41457 until we find the root cause
         # https://cartoteam.slack.com/archives/CEQLWTW9Z/p1599134417001900
         # https://app.clubhouse.io/cartoteam/story/101908/fix-encoding-error-while-logging-request
-        Rollbar.error(e)
+        # Rollbar.error(e)
```

if you want to actually silence errors, you have to turn the `log_error` above into a `log_warning` at line 180

https://github.com/CartoDB/cartodb/pull/16190/files#diff-85c80f4b786ac68618c0598445c7712a3e622b9ff1eb4d31ef60eb2b29674256R180

`log_error` logs to both stdout/err and rollbar

---

You're right, I missed that

Thanks!